AK/Vector.h

```diff
@@ -250,7 +250,7 @@ class Vector {
     // NOTE: Vector::is_null() exists for the benefit of String::copy().
     bool is_null() const { return false; }
     bool is_empty() const { return size() == 0; }
-    size_t size() const { return m_size; }
+    [[gnu::always_inline]] inline size_t size() const { return m_size; }
```

Comment:
With the 888e35f0fed393e8496414e5c285cf4ca4788f67 commit, this is now available as the `ALWAYS_INLINE` macro from `AK/Platform.h`

Reply:
I'd smite that macro to depths of git history if this codebase was mine. grrrrr.

I'll try to remember and change them, along with the stuff in bigint